### PR TITLE
:art: style: update listings, fix og:image

### PR DESCRIPTION
**Pull request by larwaa**
## Proposed Changes

- Restyle listings index page to bring it more in-line with the rest of the site
- Fix `og:image` with appropriate fallbacks
- Utilize `next/future/image` and enable S3 as asset domain
Screenshot 2022-08-24 at 17 50 23
Screenshot 2022-08-24 at 17 52 10

## Types of Changes

- [ ] New feature
- [x] Bug fix
- [ ] Enhancement/optimization
- [ ] Refactor
- [x] Style
- [ ] Build/dependencies
- [ ] Other

## Issue(s) fixed or closed by this PR

- Closes #

## Checklist

- [ ] I have added tests to cover my changes (for features/bugs)
- [x] I am pleased with the readability of the code (if not, state where you need input)
- [x] I have tested the changes and verified that they work and do not break anything (to the best of my ability)

### CODE REVIEW

File: frontend/src/pages/listings/[...listingId].tsx [hunk 1]
```diff
@@ -54,93 +47,48 @@ const ListingPage: NextPageWithLayout<InferGetServerSidePropsType<typeof getServ
...
-                position: "sticky",
-                bottom: 0,
-                padding: (theme) => theme.spacing(2),
-                zIndex: (theme) => theme.zIndex.snackbar,
-              }}
-            >
-              <Grid container direction="row" justifyContent="space-between" alignItems="center">
-                {data.listing.readMoreUrl && (
-                  <Grid item xs>
-                    <Link passHref href={data.listing.readMoreUrl}>
```
Comment (hermannm) [inline frontend/src/pages/listings/[...listingId].tsx:109]: Seems like we've dropped the `readMoreUrl` entirely?

Comment (hermannm) [reply frontend/src/pages/listings/[...listingId].tsx:109]: Hard to say where it should go now, now that the "Apply here" button is where it used to be... although I really like the new design. Could the two buttons perhaps be side by side? Only if the listing has its `readMoreUrl` field set, of course.

Comment (larwaa) [reply frontend/src/pages/listings/[...listingId].tsx:109]: Added it back, simply by adding a link to the organization's name, not the most visible, but it gets the job done.

Comment (hermannm) [reply frontend/src/pages/listings/[...listingId].tsx:109]: I like that!

**Review by hermannm — APPROVED** [review]
LGTM!

**Pull request merged**